packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/zoe-full-upgrade/zcf-upgrade-permit.json

```
@@ -0,0 +1 @@
+true
```

That's a blank check for `zcf-upgrade-permit.json` to use all powers in the bootstrap space. We neglected to go back and fill in the actual amount. Based on what `upgradeZoeAndZcf` uses from its argument, that seems to be:

```suggestion
{
  "consume": { "vatStore": true, "vatAdminSvc": true }
}
```

---

done